scripts/update_freezer_location.py

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env python
+from EPPs.common import step_argparser, StepEPP
+from EPPs.config import load_config
+
+
+class UpdateFreezerLocation(StepEPP):
+    def _run(self):
+
+        samples = self.samples
+
+        for sample in samples:
+            sample.udf['Freezer'] = self.process.udf.get('New Freezer Location')
+            sample.udf['Shelf'] = self.process.udf.get('New Freezer Location')
+            sample.udf['Box'] = self.process.udf.get('New Freezer Location')
+            sample.put()
```

Comment:
Do we really want to set all 3 UDFs to exactly the same value?

Reply:
Yes, they will all have their current freezer location removed and will then state "Sample Destroyed" or something to this effect.

Reply:
Script renamed to "remove_from_freezer.py"